### ISSUE TITLE
PEP 541 Request: PROJECT_NAME PEP 541 Request: PROJECT_NAME

### ISSUE THREAD

**Issue by dennys246** [pypi/support#5574]
### Project to be claimed

`neuronet`: https://pypi.org/project/neuronet
(Potential conflict) 'neuronetlib': https://pypi.org/project/neuronetlib/

### Your PyPI username

`dennys246`: https://pypi.org/user/dennys246

### Reasons for the request

Hi I hope you are doing well. My name is Denny Schaedig and I am a NeuroImaging Engineer at Washington University in St. Louis and I'm about to publish an research article on [an easy to use tensorflow wrapper for building and training AI on fMRI data](https://github.com/dennys246/NeuroNet) that I built. I'm hoping many Neuroscientist will use in the future so I've been trying to come up with a new name because my old tool name (KleinNet) doesn't really give people an idea of what it does and isn't incredibly memorable. I thought NeuroNet was perfect since it's analyzing NeuroImaging data with neural networks. Looking through pypi it appears this name isn't taken either, nor does it seem to be a core library used in python, but when I try and upload it with twine it fails with error 400 which appears to mean someone has taken the name?

The only library that comes close to the name is neuronetlib(https://pypi.org/project/neuronetlib/) which seems like it's not in use since it's on version 0.0.1 and the [github repo](https://github.com/skehrli/neuronetlib) associated hasn't been updated in 3 years. I reached out to them (https://github.com/skehrli/neuronetlib/issues/2) just in case they might be open to transferring over the name although I only reached out a few minutes ago so I definitely don't expect a response from them for a bit. I mostly wanted to reach out to you and see if the error truly is because neuronetlib is too close to neuronet or if there's another reason the name cannot be taken and I'm barking up the wrong tree? 

I can definitely resubmit this in a bit when I've given Sascha some time to respond to my inquire if it truly is too close to neuronetlib and not allow for another reason. Thanks for your time and I hope you have a great day!

### Maintenance or replacement?

Replacement

### Source code repositories URLs

Current PyPi library: https://pypi.org/project/neuronetlib/
Current GitHub Repo: https://github.com/skehrli/neuronetlib

Replacement GitHub Repo: https://github.com/dennys246/NeuroNet

### Contact and additional research

I only recently started researching neuronetlib but it appears the github repo and pypi library for this project are both 3 years old without any maintenance on them. I have contacted them via github and email, along with giving them my phone number to call me however I only reached out a few minutes ago and at this stage I'm more curious if neuronetlib is the reason for the error 400. Will do an analysis on traffic to see if the library is not in use in the meantime, thank you for your help with this!

### Code of Conduct

- [x] I agree to follow the PSF Code of Conduct

**Comment by Thespi-Brain** [comment]
Hi @dennys246,

The reason you were unable to use the name **neuronet** is not because of the neuronetlib project, but because **neuronet** is among a list of prohibited package/project names for PyPI. A name is considered "prohibited" if the names have been associated with malware, dependency confusion or other security risks. We also use this label for package names that are similar to existing library names or other dependencies.

We are working on coming up with an efficient process on how to release these names. Thanks for your patience!

**Comment by dennys246** [comment]
Hey @Thespi-Brain thank you for letting me know! I had a hunch that this might have been the case, but thank you for taking the time to confirm. I hope you have a wonderful week!
